dotnet/src/Microsoft.AutoGen/AgentChat/Agents/CodingAssistantAgent.cs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// CodingAssistantAgent.cs
+
+using Microsoft.AutoGen.AgentChat.Abstractions;
+using Microsoft.Extensions.AI;
+
+using ChatMessage = Microsoft.AutoGen.AgentChat.Abstractions.ChatMessage;
+using CompletionChatMessage = Microsoft.Extensions.AI.ChatMessage;
+
+namespace Microsoft.AutoGen.AgentChat.Agents;
+
+// TODO: Replatfrom this on top of AssistantAgent
+public class CodingAssistantAgent : ChatAgentBase
```

Comment:
This class potentially is going to be depreciated. Consider removing it from the PR?